### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by filippsudanov** [itdaniher/WeDoMore#13]
1) I would suggest to join information from INSTALL.rst into README.rst's Install section, since it's a necessary installation step. 

2) Does WeDoMore works on Windows/Mac platforms?

3) Technically it's possible to connect several WeDo hubs to one computer (Lego software supports up to 3 hubs). If this is possible with WeDoMore, how this can be done?

4) Similarly, one can connect two sensors, e.g. two distance sensors to one WeDo hub (also supported by Lego software). Is it possible to address them separately?

**Comment by itdaniher** [comment]
1. Seems reasonable, a PR on this would be appreciated.

2. Windows would require some platform-specific steps, but as PyUSB is supported on Windows, it shouldn't be hard. As for Mac, full functionality is slightly harder, as there's something I recall about HID devices not being unbindable from the kernel. Some details are present in #8 which would be relevant - the PR needs to be cleaned up some before I can merge it, and it'd be nice to have support for more than just the motor modules.

3. It should be trivial to do on Linux, with some hacking on the WeDoMore source code. No clue on other platforms.

4. See above.

TL;DR I don't have a WeDo anymore (I wrote this originally with borrowed hardware) and I have already spent too much of my life debugging USB issues on OSX and Windows with @kevinmehall et al. I'm open to pull requests, and happy to do more active dev on Linux functionality (3&4) if someone wants to supply me with a few WeDos and sensors.
